tests/history/conftest.py

```diff
@@ -0,0 +1,43 @@
+import shutil
+from subprocess import DEVNULL, run
+from time import perf_counter
+
+import lamindb_setup as ln_setup
+import pytest
+from lamin_utils import logger
+from laminci.db import setup_local_test_postgres
+
+AUTO_CONNECT = ln_setup.settings.auto_connect
+ln_setup.settings.auto_connect = False
+
+import lamindb as ln
+
+
+def pytest_sessionstart():
+    t_execute_start = perf_counter()
+
+    ln_setup._TESTING = True
+    pgurl = setup_local_test_postgres()
+    ln.setup.init(
+        storage="./default_storage_unit_core",
+        modules="bionty",
+        name="lamindb-unit-tests-core",
```

Comment:
This should have different `name` and `storage` to avoid clashes, these are already used by unit-core.

Reply:
This should be an entirely local test suite; there is no need to to hit cloud resources as it's self-contained.